packages/view-types/feature-list/src/selectable-table/FeatureListOptions.js

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import { OptionsContainer, OptionSelect, usePlotOptionsStyles } from '@vitessce/vit-s';
+import { TableCell, TableRow, Checkbox } from '@material-ui/core';
+import { FEATURELIST_SORT_OPTIONS, FEATURELIST_SORT_KEYS } from './constants.js';
+
+
+export default function FeatureListOptions(props) {
+  const {
+    children,
+    featureListSort,
+    setFeatureListSort,
+    featureListSortKey,
+    setFeatureListSortKey,
+    showFeatureTable,
+    setShowFeatureTable,
+  } = props;
+
+  function handleFeatureListSortChange(event) {
+    setFeatureListSort(event.target.value);
+  }
+
+  function handleFeatureListSortKeyChange(event) {
+    setFeatureListSortKey(event.target.value);
+  }
+
+  function handleShowTableChange(event) {
+    setShowFeatureTable(event.target.checked);
+  }
+
+  const classes = usePlotOptionsStyles();
+
+  return (
+    <OptionsContainer>
+      {children}
+      <TableRow>
+        <TableCell className={classes.labelCell} htmlFor="feature-list-sort-option-select">
+          Sort Ordering
+        </TableCell>
+        <TableCell>
+          <OptionSelect
+            className={classes.select}
+            value={featureListSort}
+            onChange={handleFeatureListSortChange}
+            inputProps={{
+              id: 'feature-list-sort-option-select',
+            }}
+          >
+            {FEATURELIST_SORT_OPTIONS.map(option => (
+              <option key={option} value={option}>{option}</option>
+            ))}
+          </OptionSelect>
+        </TableCell>
+      </TableRow>
+      <TableRow>
+        <TableCell className={classes.labelCell} htmlFor="feature-list-sort-key-select">
+          Sort Key
+        </TableCell>
+        <TableCell>
+          <OptionSelect
+            className={classes.select}
+            disabled={featureListSort === 'original'}
+            value={featureListSortKey}
+            onChange={handleFeatureListSortKeyChange}
+            inputProps={{
+              id: 'feature-list-sort-key-select',
+            }}
+          >
+            {FEATURELIST_SORT_KEYS.map(option => (
+              <option key={option} value={option}>{option}</option>
+            ))}
+          </OptionSelect>
+        </TableCell>
+      </TableRow>
+      <TableRow>
+        <TableCell className={classes.labelCell}>
+          Show Alternate IDs
+        </TableCell>
+        <TableCell className={classes.inputCell}>
+          <Checkbox
+            className={classes.checkbox}
+            checked={showFeatureTable}
+            onChange={handleShowTableChange}
+            name="feature-list-show-table"
+            color="default"
+          />
+        </TableCell>
+      </TableRow>
```

Comment:
```suggestion
      {hasFeatureLabels ? (
        <>
          <TableRow>
            <TableCell className={classes.labelCell} htmlFor="feature-list-sort-key-select">
              Sort Key
            </TableCell>
            <TableCell>
              <OptionSelect
                className={classes.select}
                disabled={featureListSort === 'original'}
                value={featureListSortKey}
                onChange={handleFeatureListSortKeyChange}
                inputProps={{
                  id: 'feature-list-sort-key-select',
                }}
              >
                {FEATURELIST_SORT_KEYS.map(option => (
                  <option key={option} value={option}>{option}</option>
                ))}
              </OptionSelect>
            </TableCell>
          </TableRow>
          <TableRow>
            <TableCell className={classes.labelCell}>
              Show Alternate IDs
            </TableCell>
            <TableCell className={classes.inputCell}>
              <Checkbox
                className={classes.checkbox}
                checked={showFeatureTable}
                onChange={handleShowTableChange}
                name="feature-list-show-table"
                color="default"
              />
            </TableCell>
          </TableRow>
        </>
      ) : null}
```

We can use the `hasFeatureLabels` prop from the above suggestion to decide whether to render these options

Reply:
Good point. I wrote it as `hasFeatureLabels && ()` rather than `hasFeatureLabels ? (): null` to avoid rendering an extra null in the cases where `hasFeatureLabels` is false.